hedera-mirror-rest/viewmodel/networkNodeViewModel.js

```diff
@@ -17,7 +17,9 @@
 import AddressBookServiceEndpointViewModel from './addressBookServiceEndpointViewModel';
 import EntityId from '../entityId';
 import * as utils from '../utils';
+import {getIncludeAdminKey} from '../config';
 
+const includeAdminKey = getIncludeAdminKey();
```

Comment:
I don't think we need one function per property.

```suggestion
import config from '../config';

const includeAdminKey = config.response.includeAdminKey;
```